cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SubqueryExpressionsTest.scala

```diff
@@ -140,5 +144,107 @@ class SubqueryExpressionsTest extends DocumentingTest {
         }
       }
     }
+
+    section("COUNT subqueries", "count-subqueries") {
+      p("An count subquery can be used to find out how many times a specified pattern appears in the data.")
```

Comment:
```suggestion
      p("A `COUNT` subquery can be used to find out how many times a specified pattern appears in the data.")
```
Here you call it "pattern".